schema/user.schema.graphql

```diff
@@ -68,10 +68,9 @@ type User @entity {
   bitcoinTransfers: [BitcoinTransfer!] @derivedFrom(field: "user")
   fastBTCBridgeStats: [FastBTCBridgeStat!] @derivedFrom(field: "user")
   """
-  Cross ChainBridge transfers from RSK and to RSK
+  EVM Bridge transfers
```

Comment:
rskUser is here so the user can be correctly linked to their bridge transfers. Otherwise the derived list of cross transfers is incorrect.

I think it's more of a bug fix than a refactor

Reply:
for incoming transfers I am not sure the transaction initiator is the actual user but rather is the federator

Reply:
ok, I'll check

Reply:
For incoming, "user" is the receiver and "from" is the federator. 
For outgoing, user is "from" and "sender" is the federator.
I propose we change the schema to rskUser and externalChainUser
If you think it's necessary to also store the federator address, it should be under a property called "federator" for clarity

Reply:
I don't think that the federator is needed in this context
but why change sender/receiver property names?

Reply:
So the rskUser can be a foreign key to User and User entities aren't created for non-rsk addresses (eg a User doesn't get created for the ETH address an rskUser is bridging to)

Reply:
Is this ok or are there still issues?

Reply:
I am pretty sure I fixed that already

Reply:
Where?